.circleci/config.yml

```diff
@@ -240,6 +273,16 @@ jobs:
           command: npm test
 
 workflows:
+  release_storybook:
+    jobs:
+      - release_storybook:
+          context: binary-frontend-artifact-upload
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^storybook.*/
+
   release_staging:
     jobs:
       - release_staging:
```

Comment:
![image](https://user-images.githubusercontent.com/59427711/125385327-db7a7000-e3cc-11eb-9afd-f31914499ff1.png)
both of these requires context otherwise `DOCKER` secrets wont be available during build

Reply:
We're using `Environment Variables` in Circleci also.